samples/BoltOn.Samples.WebApi/Startup.cs

```diff
@@ -32,12 +32,17 @@ public void ConfigureServices(IServiceCollection services)
             });
 
             services.AddDbContext<SchoolDbContext>(options =>
-             {
-                 options.UseSqlServer("Data Source=127.0.0.1;initial catalog=Testing;persist security info=True;User ID=sa;Password=$Password1;");
-             });
+            {
+                options.UseSqlServer("Data Source=127.0.0.1;initial catalog=Testing;persist security info=True;User ID=sa;Password=$Password1;");
+            });
 
-            services.Configure<CosmosDbSettings>(options => Configuration.GetSection("CosmosDbSettings").Bind(options));
-            services.AddSingleton<CollegeDbContext>();
+            //services.Configure<CosmosDbSettings>(options => Configuration.GetSection("CosmosDbSettings").Bind(options));
+            services.AddCosmosDbContext<CollegeDbContext>(new CosmosDbConfiguration
```

Comment:
rename the class to CosmosDbOptions, and let the extension method take `Action<CosmosDbOptions>` like the first extension here https://github.com/gokulm/BoltOn/blob/master/src/BoltOn/Extensions.cs.

Reply:
@gokulm : gottcha updated.